src/testRunner/unittests/tscWatch/consoleClearing.ts

```diff
@@ -2,8 +2,8 @@ namespace ts.tscWatch {
     describe("unittests:: tsc-watch:: console clearing", () => {
         const currentDirectoryLog = "Current directory: / CaseSensitiveFileNames: false\n";
         const fileWatcherAddedLog = [
-            "FileWatcher:: Added:: WatchInfo: /f.ts 250 Source file\n",
-            "FileWatcher:: Added:: WatchInfo: /a/lib/lib.d.ts 250 Source file\n"
+            "FileWatcher:: Added:: WatchInfo: /f.ts 250 undefined Source file\n",

```

Comment:
What does `undefined` refer to here?

Reply:
undefined is watchOptions passed to the file watcher